packages/editor-web-component/package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "@serlo/editor-web-component",
+  "version": "0.0.1",
+  "homepage": "https://de.serlo.org/editor",
+  "bugs": {
+    "url": "https://github.com/serlo/frontend/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/serlo/frontend.git"
+  },
+  "license": "Apache-2.0",
+  "author": "Serlo Education e.V.",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./dist/serlo-editor-web-component.js",
+      "require": "./dist/serlo-editor-web-component.js",
+      "types": "./dist/editor-web-component.d.ts"
+    }
+  },
+  "main": "./dist/serlo-editor-web-component.js",
+  "module": "./dist/serlo-editor-web-component.js",
+  "types": "./dist/editor-web-component.d.ts",
+  "files": [
```

Comment:
this leads to a lot of things being exposed through the package under the dist directory. Common practice would be exposing those types via tree shakable semantic subpaths and strip internals. 

![image](https://github.com/serlo/frontend/assets/1191486/7bd579e5-6897-4f1f-bb49-73cbedcd7f11)

Understandable if a semantic structure won't be part of the 1.0 release since tree shaking would really not bring any benefit at the moment and would mostly just bloat the build process atm. I'm bringing this up because it's usually good practice to have well structured import paths - my suggestion for now would be just not exposing the dist directory in the final bundle but rather only the index export (ln 15). Not a hugely important topic.